src/content/support/p1-process/index.md

```diff
@@ -0,0 +1,90 @@
+
+
+## P1 Incident Process
+
+The process starts from the well-known [Incident Command System](https://en.wikipedia.org/wiki/Incident_Command_System) used by US firefighters to manage emergency situations. Obviously we have adapted to manage developer platforms. 
+
+Our main tenet is to have a simple process integrated with our incident tooling ([Incident.io](https://incident.io/)) to simplify the life of our engineers. Once a critical incident is declared none wants to read a process but be driving by it. 
+
+The process can be broken down in these steps:
+
+1. [Identify](#identify)
+2. [Investigate](#investigate)
+3. [Fixing](#fixing)
+4. [Monitoring](#monitoring)
+5. [Closing up](#closing-up)
+
+### Identify
+
+First step is to identify the incident and understand the severity. In our case, there are three possible sources:
+
+- Alert received pointing to a impaired production system
+- Customer sent us a urgent email 
+- Customer reached us via Slack
+
+Once our engineer acknowledge the message we need to confirm the severity to trigger the P1 process. In case the customer paged us, the Giant Swarm engineer joins the call or thread to verify the severity. In case an alert paged us, the engineer connects to the platform and measure the impact.
+
+Once the criticality is confirmed we continue the workflow by [declaring an Incident](https://help.incident.io/en/articles/5947915-declaring-incidents), which creates a Slack incident channel and triggers a custom [workflow](https://help.incident.io/en/articles/6971329-getting-started-with-workflows) that drives the engineer through the next steps. 
+
+When the incident is created there are some data the engineer must fill in (short summary, severity and customer impacted). When severity is `Critical` (P1 incident) the first thing to do is building the team. Most of the times the engineer creating the channel is not part of the [Incident Coordinators Group]() so he needs to escalate it to get someone from that team. For that reason, when you create an incident channel, incident.io show you a button to escalate. In the end, we need at least two person team to manage a critical incident (communications and operations).
+
+### Investigate
+
+Once team is built the person assigned to the Operations Engineer role will carry on with the investigation. The Incident Coordinator will be in contact with the customer, via messaging or in a call, and will provide information to the Operator to help with the investigation.
+
+We leave space to the Operations Engineers to focus on the investigation but we establish 20 minutes periods to get back and inform on the state. Most of the times the OEs share the findings in the channel and IC pin those message to help tracking the actions performed.
+
+In case the Incident Coordinator realize we need to grow the group, he can always escalate to more memebers of the team using incident.io commands.
```

Comment:
```suggestion
In case the Incident Coordinator realize we need to grow the group, he can always escalate to more members of the team using incident.io commands.
```